src/Service/S3/src/ValueObject/CompletedPart.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace AsyncAws\S3\ValueObject;
+
+class CompletedPart
+{
+    /**
+     * Entity tag returned when the part was uploaded.
+     */
+    private $ETag;
+
+    /**
+     * Part number that identifies the part. This is a positive integer between 1 and 10,000.
+     */
+    private $PartNumber;
+
+    /**
+     * @param array{
+     *   ETag?: null|string,
+     *   PartNumber?: null|int,
+     * } $input
+     */
+    public function __construct(array $input)
+    {
+        $this->ETag = $input['ETag'] ?? null;
```

Comment:
Should we somehow verify the `Etag` that is is quoted? Or dont we care?

Reply:
how does other SDK works?

Reply:
From what I can see, the official API do nothing at all regarding ETags. I dont think we should either